hardhat.config.ts

```diff
@@ -0,0 +1,66 @@
+import "hardhat-gas-reporter"
+import "solidity-coverage";
+import "@nomicfoundation/hardhat-toolbox"
+import { config as dotenvConfig } from "dotenv"
+import { HardhatUserConfig } from "hardhat/config"
+import { NetworksUserConfig } from "hardhat/types"
+import { resolve } from "path"
+import { config } from "./package.json"
+
+dotenvConfig({ path: resolve(__dirname, "./.env") })
+
+function getNetworks(): NetworksUserConfig {
+  if (!process.env.ALCHEMEY_KEY)
+    throw new Error(
+      `ALCHEMEY_KEY env var not set. Copy .env.template to .env and set the env var`
+    )
+  if (!process.env.MNEMONIC)
```

Comment:
I don't think it should be strictly necessary to provide a mnemonic to use the repo

Reply:
I think if there's no mnemonic you should not pass the accounts argument in the hardhat settings.